example/assessments/iama.yaml

```diff
@@ -0,0 +1,20 @@
+name: IAMA
```

Comment:
Is is accidental that the name and the remarks are not explicit strings?

Reply:
it is not required to make explicit strings